rfcs/0000-community-coordination-hub.md

```diff
@@ -0,0 +1,158 @@
+---
+feature: community_coordination_hub
+start-date: 2020-07-30
+author: David Arnold <dar@xoe.solutions>
+co-authors: (find a buddy later to help out with the RFC)
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: (will contain links to implementation PRs)
+---
+
+# Summary
+[summary]: #summary
+
+Adopt goals, spirit, concept and execution of [`kubernetes/community`]
+where it _does_ fit [us](https://nixos.org/).
+
+[`kubernetes/community`]: https://github.com/kubernetes/community
+
+# Motivation
+[motivation]: #motivation
+
+We lack an optimum level of _empowerment_ on distinct vital aspects for
+the best [leverage](https://nav.al/least) on the available resources:
+
+- Efficient Management Processes, that bias the whole community towards 
+  _coordinated and optimal Resource Employment_.
+- Organizational Structure, that enacts necesary degrees of _Division of Labor_ 
+  and _Coordination_.
+- Organizational Culture, that helps to overcome the complexity bias. ([KISS](https://en.wikipedia.org/wiki/KISS_principle))
+
+Analysis inspired by [New St.Gallen Managament Model].
+
+[New St.Gallen Managament Model]: https://cio-wiki.org/wiki/The_New_St._Gallen_Management_Model
+
+# Detailed design
+[design]: #detailed-design
+
+A repository shall exist, named `NixOS/community`, which acts as central hub
+for coordinating the community dynamics.
+
+It shall comprehend, consolidate and showcase aspects of:
+
+- Communication
+  - Channels
+  - Modes & Policies
+- Governance
+  - Processes
+  - Structure
+- Onboarding
+  - Code Contributions
+  - Community Engagement
+
+It shall omit any technical subject about nix or code in general, other than
+repository tooling.
+
+To implement, a detailed review of https://github.com/kubernetes/community
+shall be conducted as joint effort between the interested parties and as 
+an Annex I to this RFC.
+
+Based on this review, a clean repository shall be enacted implementing
+such aspects considered worth porting to our community.
+
+As a general guideline for this review, the vast success and impressive
+dynamics of the kubernetes ecosystem shall be acknowledged not only as a
+proof of concept, but as a very sucessfull and inspiring [viable system].
```

Comment:
@blaggacao again, I'd like not to wear you out, but I don't think we need to use high words such as "viable system" to express the idea. My philosophy is that good ideas are simple. So perhaps:

```suggestion
As a general guideline for this review, we'd acknowledge the success
and impressive dynamics of the Kubernetes ecosystem.
```

And don't get me wrong, I'm sure that people have put out a lot of research through out history to these fields. But, for the common being, practical and simple suggestions should sprout out of the research.